packages/perseus/src/__tests__/renderer-api.test.tsx

```diff
@@ -215,10 +215,12 @@ describe("Perseus API", function () {
         });
     });
 
-    describe("widget placeholders", function () {
-        it("should replace widgets with [WIDGET]", function () {
+    describe("placeholders", function () {
+        it("should replace widgets with widgetPlaceholder", function () {
             const widgetPlaceholder = (
-                <span data-testid="widget-container">[WIDGET]</span>
+                <span data-testid="widget-container" key="1">
```

Comment:
The lack of `key` on these elements was causing a React warning because widgets are rendered as a list of children (even when there's only 1).